### DISCUSSION TITLE
Railway部署成功后，微信扫log里二维码，总是显示”Service unavailable for this account“

### DISCUSSION THREAD

**Discussion by leo-usa** [zhayujie/chatgpt-on-wechat#1329] [Q&A]
我在Railway上部署成功了，然后到log detail里查看二维码，扫了很多次都无法成功。我是用的海外版的weChat。搜了issues里面也没有对类似问题的解答。有人现在Railway的部署还工作吗？

**Comment by jianghongping** [comment]
你的微信账号未实名，或者注册手机号不是国内的。
